index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
+    <meta name="" content="">
+    <link rel="stylesheet" href="styles.css" title="" type="text/css" />
+    <title></title>
+  </head>
+  <body >
+    <div id="headline">
+    <div id="intro"><p id="greetings">Hey There, I am Solomon</p></div>
+    <div id="job"><p id="my_job">I am a Software Developer</p></div>
+    <div id="description">
+     <p id="ability"> I can help you build a product, website or feature. Look through some of my work and experience!. If you like what you see and have a project you need coded, don't hesitate to contact me.
+     </p>
+    </div>
+    <ul id="container">
+      <li class="social">
+        <a href="https://github.com/Solobarine">
+          <img src="/Image/social media/github_1.png" alt="my github handle" width="20" height="20"></a>
```

Comment:
- [ ] [OPTIONAL] Please change your image path from ` src="/Image/social media/github_1.png" ` to ` src="./Image/social media/github_1.png" `